wazuh/wazuh-dashboard-plugins
# Add link to agent id and name in discover table

### Description
Adds a link to the agent's id or name in the discover tables when the agent is not 000, and resolves a console error.
 
### Issues Resolved

- #6672 

### Evidence

Agent 000

<img width="1031" alt="image" src="https://github.com/wazuh/wazuh-dashboard-plugins/assets/63758389/86774c89-cc80-482a-b645-1c3a37cf16aa">

Other Agent

<img width="1042" alt="image" src="https://github.com/wazuh/wazuh-dashboard-plugins/assets/63758389/d47ddd01-0beb-4685-a955-af7fb67ab659">

Console error

<img width="1511" alt="image" src="https://github.com/wazuh/wazuh-dashboard-plugins/assets/63758389/c71cf56d-8a84-4c9e-ab31-5777bd0e7182">

### Test

1. Navigate to 'some dashboard/Events'
2. Add agent name and id to the table
3. [ ] In the agent `000` the name and id must be a plain text
4. [ ] The other agents in the name and ID must be a link to the agent view.
5. [ ] In the agent view, the `intl` console error should not appear.

### Check List
